public/src/views/FormView.jsx

```diff
@@ -107,7 +116,13 @@ export const FormView = () => {
           </IconographicLabel>
           <FontAwesomeIcon icon={faSun} />
         </CheckboxButton>
-        <PrimaryButton role="button" onClick={submit}>Submit</PrimaryButton>
+        <PrimaryButton
+          disabled={!pee && !poo && !accident && !sleep && !awoke}
```

Comment:
What a fun line of code to read.

You could consider:
`[pee, poop, accident, sleep, awoke].some(Boolean)`

! && ! && ! && becomes easy to overlook forgetting a !

Reply:
Neat! Learned something new. Fixed.